Assets/Mariana/Project 3/Scripts/MovementRight.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Mariana
+{
+    public class MovementRight : MonoBehaviour
+    {
+        //This script controls the right leg
+        private Rigidbody2D rightLeg;
+        public float speedLeft;
```

Comment:
When I loaded the project these values were set to 0 in the inspector. Did you know you can set defaults with

`public float speedLeft = 10f;`

That can help you get things set up correctly. In this case, it's because the left leg wasn't a prefab so the value was zero when I tested.